### PR TITLE
Load packages less often

### PR DESCRIPTION
**Pull request by StachuDotNet**
No changelog

Works for me - what you were expecting, @pbiggar ?

### CODE REVIEW

**Comment by pbiggar** [comment]
reload_packages can go too, right?

**Comment by StachuDotNet** [comment]
you're right - I forgot `./scripts/build/reload-packages ` was run as part of `run-backend-server`. (it's higher up in the script so I missed it)

**Comment by StachuDotNet** [comment]
this is good afaict, but CI is a bit broken for me

File: scripts/run-backend-server [hunk 1]
```diff
@@ -108,6 +108,7 @@ echo "Running queueworker"
 echo "Running bwddangerserver"
 "${BWDDANGERSERVER_EXE}" > "$LOGS/bwddangerserver.log" 2>&1 &
 
+echo "Loading packages into dark canvas"
```
Comment (pbiggar) [inline scripts/run-backend-server:111]: AFAIR, I moved this `echo` into the script.

**Comment by pbiggar** [comment]
Nice.

**Pull request merged**